src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -41,14 +47,27 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
   );
 
   return (
-    <div className={posterClassnames} onClick={handleEnterVenue}>
-      <p className="PosterPreview__title">
-        {posterVenue.name}: {title}
-      </p>
+    <div className={posterClassnames}>
+      <a href={posterAbsUrl} target="_ohbm-poster-preview">
+        📄
+      </a>
+      {posterVenue.iframeUrl?.trim() !== "" ? (
+        <a href={posterVenue.iframeUrl} {...externalUrlAdditionalProps}>
+          {" "}
+          ⬇{" "}
+        </a>
+      ) : (
+        <div />
+      )}
```

Comment:
move `posterVenue.iframeUrl?.trim() !== ""` to a const. eg. const hasIframeUrl.

then it'll look like

`hasIframeUrl && <a></a>`

Reply:
Thanks, will do.

But in general you don't oppose such a solution?

Reply:
A hint on how to tune css (see original description on top) to line them up horizontally would be  appreciated.

Reply:
No, I don't, except for, I'd still leave the `onClick` handler on the whole poster. And added the `stopPropagation` for both icons' handlers

Reply:
note: for a proper url, needs #1534 , but may be that PR can just supersede this one altogether.